x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/config/AnalysisConfig.java

```diff
@@ -133,7 +135,7 @@
     private final List<Detector> detectors;
     private final List<String> influencers;
     private final Boolean multivariateByFields;
-    private final TimeValue modelPruneWindow;
+    private TimeValue modelPruneWindow;
```

Comment:
This class is intended to be immutable, and code that uses it will be assuming this.  So this field will have to be changed back to `final`.